airflow/decorators/__init__.pyi

```diff
@@ -41,6 +42,7 @@ __all__ = [
     "task_group",
     "python_task",
     "virtualenv_task",
+    "preexisting_virtualenv_task",
```

Comment:
Minor English language nit: while "pre-existing virtual env" is perfectly understandable, the "pre" prefix is unnecessary, "existing_virtualenv_task" has the same meaning and is slightly shorter.

Reply:
I think I found a better name. Stay tuned.